cfgov/jinja2/v1/document-detail/index.html

```diff
@@ -103,7 +104,23 @@
     {# TODO: Add Related Metadata and Topics Sidebar molecules. #}
     {% if prototype %}
         {{ related_posts.render(global_dict.related_posts_settings) }}
-
+        {{ related_metadata.render([{
+                'title': 'Metadata Label 1',
+                'type' : 'date',
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5db4db6d035790fefeaa840c4830e2f7d9629292